File: WinForms/VB/LayoutMainDemo/Modules/XtraLayoutPanel/Stack/LoginView.vb
Imports System.ComponentModel
Imports System.Drawing
Imports DevExpress.Utils.Layout

Namespace DevExpress.XtraLayout.Demos.Modules.XtraLayoutPanel

    Public Partial Class LoginView
        Inherits XtraLayoutPanelDemoViewBase

        Public Sub New()
            InitializeComponent()
        End Sub

        Public Overrides ReadOnly Property LayoutPanel As XtraLayoutPanelBase
            Get
                Return stackPanelField
            End Get
        End Property

        Public ReadOnly Property StackPanel As StackPanel
            Get
                Return stackPanelField
            End Get
        End Property
    End Class
End Namespace

File: WinForms/CS/LayoutMainDemo/Modules/XtraLayoutPanel/Stack/LoginView.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.Utils.Layout;
using DevExpress.XtraVerticalGrid;

namespace DevExpress.XtraLayout.Demos.Modules.XtraLayoutPanel {
    public partial class LoginView : XtraLayoutPanelDemoViewBase {
        public LoginView() {
            InitializeComponent();
        }
        public override XtraLayoutPanelBase LayoutPanel { get { return stackPanel; } }
        public StackPanel StackPanel { get { return stackPanel; } }
    }
}
